### PR TITLE
Allow to clear selected tags on list pages

### PR DESCRIPTION
**Pull request by kravets-levko**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [x] Feature

## Description

Show additional item on Queries and Dashboard lists that allows to unselect all tags and show all queries/dashboards.

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

image
image

### CODE REVIEW

**Comment by kravets-levko** [comment]
@gabrieldutra Probably it will be easier to review commits one by one - I tried to keep changes logically isolated

File: client/app/pages/dashboards/DashboardList.jsx [hunk 1]
```diff
@@ -95,7 +95,12 @@ class DashboardList extends React.Component {
                 onChange={controller.updateSearch}
               />
               <Sidebar.Menu items={this.sidebarMenu} selected={controller.params.currentPage} />
-              <Sidebar.Tags url="api/dashboards/tags" onChange={controller.updateSelectedTags} />
+              <Sidebar.Tags
+                url="api/dashboards/tags"
+                onChange={controller.updateSelectedTags}
+                showUnselectAll
+                unselectAllButtonTitle="View All Dashboards"
```
Comment (kravets-levko) [inline client/app/pages/dashboards/DashboardList.jsx:102]: I'm open to suggestions for better labels (cc @leini-db)

**Comment by leini-db** [comment]
One reason we want to add something like "all tags" here is to indicate those are tags. 
i think people are confused on what are those things at first sight. 
I'm not sure "view all queries" will achieve this goal.

**Comment by arikfr** [comment]
To add to @leini-db's comment: "View All Queries" is a confusing text when you think about this list in context of the My Queries or Favorites pages.

**Comment by kravets-levko** [comment]
@arikfr I totally agree. But `All Tags` is confusing as well. So I'm open for better ideas (for both pages, of course)

**Comment by arikfr** [comment]
Maybe we should have two things here:

1. Just a title saying "Tags" or something.
2. "Clear Filter" button after selecting tags.

I will ping Wenbo for an opinion.

**Comment by wenbow** [comment]
Agree with @arikfr and @leini-db . The two ideas "Tags" title & "Clear Filter" button definitely work.

**Comment by kravets-levko** [comment]
@wenbow @arikfr @leini-db 

image
image

**Review by arikfr — APPROVED** [review]

**Pull request merged**